nemo/collections/asr/models/rnnt_models.py

```diff
@@ -130,6 +136,44 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self._optim_variational_noise_std = 0
             self._optim_variational_noise_start = 0
 
+    def _setup_loss(self, cfg: Optional[DictConfig]):
+        """
+        Helper method to extract the rnnt loss name, and potentially its kwargs
+        to be passed.
+
+        Args:
+            cfg: Should contain `loss_name` as a string which is resolved to a RNNT loss name.
+                If the default should be used, then `default` can be used.
+                Optionally, one can pass additional kwargs to the loss function. The subdict
+                should have a keyname as follows : `{loss_name}_kwargs`.
+
+                Note that whichever loss_name is selected, that corresponding kwargs will be
+                selected. For the "default" case, the "default_kwargs" will be used, not the
+                "{resolved_default}_kwargs".
+
+        Examples:
+            .. code-block:: yaml
+                loss_name: "default"
+
+                default_kwargs:
+                    kwargs1: some_val
+
+                warprnnt_kwargs:
+                    kwargs2: some_other_val
+
+        Returns:
+            A tuple, the resolved loss name as well as its kwargs (if found).
+        """
+        if cfg is None:
+            cfg = DictConfig({})
+
+        loss_name = cfg.get("loss_name", "default")
+        loss_kwargs = cfg.get(f"{loss_name}_kwargs", None)
```

Comment:
Do we need to have the name of the loss in the param?
Is there a way we use target as the class name of the loss and let hydra create the appropriate loss?